manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md

```diff
@@ -60,32 +99,77 @@ PUT _ilm/policy/my_policy
+1. On the **Logistics** page: 
+    1. Specify a name for the template.
+    1. Specify a pattern to match the indices you want to manage with the lifecycle policy. For example, `my-index-*`.
+    1. If you're storing continuously generated, append-only data, you can opt to create [data streams](/manage-data/data-store/data-streams.md) instead of indices for more efficient storage. If you enable this option, you can also enable **Data retention** to configure how long your indexed data is kept.
 
-::::{tip}
-An `index.lifecycle.rollover_alias` setting is only required if using {{ilm}} with an alias. It is unnecessary when using [Data Streams](../../data-store/data-streams.md).
-::::
+        :::{note}
+        Since you're creating an index lifecycle policy to manage indices, the  **Data retention** option should be left disabled. Data retention is applicable only if you're using a data stream lifecycle, which is an alternative to ILM. Refer to the [Data stream lifecycle](/manage-data/lifecycle/data-stream.md) to learn more.
+        :::
+
+
+    1. Configure any other options you'd like, including:
+        * The [index mode](elasticsearch://reference/elasticsearch/index-settings/time-series.md) to use for the created indices.
+        * The template priority, version, and any metadata.
+        * Whether or not to overwrite the `action.auto_create_index` cluster setting.
+        
+        Refer to the [Create or update index template API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-put-index-template) documentation for details about these options.
 
+1. On the **Component templates** page, use the search and filter tools to select any [component templates](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-component-template) to include in the index template. The index template will inherit the settings, mappings, and aliases defined in the component templates and apply them to indices when they're created.
 
-You can use the {{kib}} Create template wizard to create a template. To access the wizard, open the menu and go to **Stack Management > Index Management**. In the **Index Templates** tab, click **Create template**.
+1. On the **Index settings** page:
+    1. Configure ILM by specifying the [ILM settings](https://www.elastic.co/docs/api/doc/elasticsearch/configuration-reference/index-lifecycle-management-settings#_index_level_settings_2) to apply to the indices:
+        * `index.lifecycle.name` - The lifecycle policy to manage the created indices.
+        * `index.lifecycle.rollover_alias` - The index [alias](/manage-data/data-store/aliases.md) used for querying and managing the set of indices associated with a lifecycle policy that contains a rollover action.
```

Comment:
At this point, could we consider collapsing the existing doc into this? This has screenshots but the previous does exist & is commonly linked by Support : https://www.elastic.co/docs/manage-data/lifecycle/index-lifecycle-management/tutorial-automate-rollover